packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/workflow/Workflow.java

```diff
@@ -46,12 +42,22 @@ public class Workflow extends TerariumAsset {
 	@JdbcTypeCode(SqlTypes.JSON)
 	private List<WorkflowEdge> edges;
 
+	@OneToOne(mappedBy = "workflow")
+	@Where(clause = "deleted_on IS NULL")
+	@ToString.Exclude
+	@JsonManagedReference
+	private Simulation simulation;
```

Comment:
I think there are some miscommunication here, a workflow should not have a direct reference to a Simulation. 

The reverse is also true that a Simulation does not need a direct reference to a Workflow.